tests/by-util/test_chmod.rs

```diff
@@ -2,6 +2,8 @@
 //
 // For the full copyright and license information, please view the LICENSE
 // file that was distributed with this source code.
+#![allow(clippy::manual_assert)]
```

Comment:
why not fixing them?

Reply:
I thought the PR would be too big. I'm fixing it now.